mkt/websites/indexers.py

```diff
@@ -32,29 +33,32 @@ def get_mapping(cls):
                     'description': {'type': 'string',
                                     'analyzer': 'default_icu'},
```

Comment:
It's probably a good idea to use `'position_offset_gap': 100` for both `description` and `title` since they are a list of all translations. This helps prevent the last word of one translation and the first word of the next from matching in phrase queries.